sidm/tools/histogram.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, axes, storage="weight", evt_mask=None):
         # Allow all events to pass if no mask is explicitly provided
         self.evt_mask = (lambda objs: slice(None)) if evt_mask is None else evt_mask
         self.hist = None
-
+        
```

Comment:
Hi @yfv2ev, would you mind removing this extra whitespace?

Reply:
Should be fixed now